### PR TITLE
update falco.md

### PR DESCRIPTION
**Pull request by jiazxjason**
zh-trans falco.md

### CODE REVIEW

**Comment by tengqm** [comment]
instead of updating the translation, pls consider remove the English and Chinese version of this doc because these contents fall into "3rd party" documentation which are being dropped from the website.

File: content/zh/docs/tasks/debug-application-cluster/falco.md [hunk 1]
```diff
@@ -1,62 +1,38 @@
...
--->
-[Falco](https://falco.org/)是一个开源项目，用于为云原生平台提供入侵和异常检测。本节介绍如何设置 Falco、如何将审计事件发送到 Falco 公开的 Kubernetes Audit 端点、以及 Falco 如何应用一组规则来自动检测可疑行为。
+[Falco](https://falco.org/) is an open source project for intrusion and abnormality detection for Cloud Native platforms. This section describes how to set up Falco, how to send audit events to the Kubernetes Audit endpoint exposed by Falco, and how Falco applies a set of rules to automatically detect suspicious behavior.
 
+-->
 
+### 使用 Falco 收集审核事件
 
-<!-- body -->
+Falco 是一个用于云原生平台入侵和异常检测的开源项目。 本节介绍如何设置 Falco，如何将审核事件发送到Falco公开的 Kubernetes Audit 端点以及 Falco 如何用一组规则来自动检测可疑行为。
```
Comment (markthink) [inline content/zh/docs/tasks/debug-application-cluster/falco.md:21]: 翻译时丢失了原始网址链接..

**Comment by markthink** [comment]
@jiazxjason 需要解决文件冲突

File: content/zh/docs/tasks/debug-application-cluster/falco.md (hunk 1 above)
Comment (jiazxjason) [reply content/zh/docs/tasks/debug-application-cluster/falco.md]: 已修改

**Pull request closed without merging**